resources/lib/oojs-ui/oojs-ui.js

```diff
@@ -2728,6 +2728,9 @@ OO.ui.WindowManager.static.sizes = {
 	},
 	'800px': {
 		width: 800
+	},
+	page: {
```

Comment:
I'd consider modifying VE to be preferable. The OOjs UI library really shouldn't have anything specific to Oasis in it, and any changes to its functionality that aren't generic should go in VE modules that extend it.